test/scripts/models/contents/page.js

```diff
@@ -0,0 +1,32 @@
+describe('test page model', function () {
+  'use strict';
+  var Page,
+    expect = chai.expect;
+
+  beforeEach(function (done) {
+    require(['cs!models/contents/page'], function () {
+      Page = arguments[0];
+      done();
+    });
+  });
+  describe('model defaults', function () {
+    it('should have default type for mediaType', function () {
+      var page = new Page();
+      expect(page.get('mediaType')).to.equal('application/vnd.org.cnx.module');
+    });
+  });
+  describe('results returned', function () {
+    it('should show default text if page is loaded and content is empty', function () {
```

Comment:
This doesn't appear to test for default text, only that a page's content is what it is set to be.